lib/middleware/validate_identity.js

```diff
@@ -35,7 +42,11 @@ module.exports = function() {
       if (!identity) {
 
         log.error('Identity not found');
-        return next(new errors.UnauthorizedError('Identity not found'));
+        err = new errors.UnauthorizedError('Identity not found');
```

Comment:
why this is not ` return next(new errors.UnauthorizedError('Identity not found'));` ?